File: Projects/10/Java/Owl.java
/* Owl.java provides an Owl subclass of Bird.
 *
 * Completed by: EMily Costa
 * Date: 4/27/2020
 ******************************************************/

public class Owl extends Flyingbird {

    /* explicit constructor
     * Receive: name, a String 
     * PostCond: myName == name.
     */
    public Owl(String name) {
        super(name);
    }

    /* A Owl's class name
     * Return: a Owl's class name
     */
    public String getClassName() {
        return "Owl";
    }
   
    /* An Owl's Call
     * Return: an owl-call ("Whoo-hoo!").
     */
    public String call() {
        return "Whoo-hoo";
    }
   
   } 

File: Projects/10/Ruby/Owl.rb
# Owl.rb | Defines the Owl class which inherits attributes and methods
#   from the Bird superclass.
#
# Begun by: Dr. Adams, for CS 214 at Calvin College.
# Completed by: Emily Costa
# Date: 4/27/2020
####################################################

require './Flyingbird.rb'

class Owl < Flyingbird

    def initialize(name)
        @name = name
    end

    def className
        "Owl"
    end

    def call
        "Whoo-hoo!"
    end

end